src/index.css

```diff
@@ -24,3 +24,8 @@ body {
 .search-row input {
   margin-right: 10px;
 }
+
+/* making space between logo and h1 */
+header > img {
+  margin-right: 2rem;
+}
```

Comment:
Is there a reason to have this in the index.css? Remember for index.css this would be to change the whole project.